app/assets/stylesheets/theme/theme.scss

```diff
@@ -0,0 +1,684 @@
+//@extend-elements
+//original selectors
+//.dropdown-menu >li >a:hover, .dropdown-menu >li >a:focus, .dropdown-menu >.active >a, .dropdown-menu >.active >a:hover, .dropdown-menu >.active >a:focus
+%extend_1 {
+  background-image: linear-gradient(to bottom, #007fff 0, #0072e6 100%);
+  background-repeat: repeat-x;
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#ff007fff', endColorstr='#ff0072e6', GradientType=0);
+  background-color: #0072e6;
+  color: #ffffff;
+}
+
+//original selectors
+//.ribbon-red:before, .ribbon-red:after
+%extend_2 {
+  content: "";
+  border-top: 3px solid #6e8900;
+  border-left: 3px solid transparent;
+  border-right: 3px solid transparent;
+  position: absolute;
+  bottom: -3px;
+}
+
+
+html {
+  text-rendering: optimizeLegibility !important;
+}
+* {
+  &:before {
+  }
+  &:after {
+  }
+}
+.container {
+  max-width: 997px !important;
+}
+.btn-dropzone {
+  color: #000000;
+  font-size: 10px;
+  padding: 0 10px;
+  width: 100%;
+  &:hover {
+    background: #d2160a;
+    color: #ffffff;
+  }
+}
+.btn-file {
+  position: relative;
+  overflow: hidden;
+  font-size: 10px;
+  padding: 0 10px;
+  width: 100%;
+  &:hover {
+    background: #007fff;
+    color: #ffffff;
+  }
+  input[type=file] {
+    width: 100%;
+    position: absolute;
+    top: 0;
+    right: 0;
+    min-width: 100%;
+    min-height: 100%;
+    font-size: 999px;
+    text-align: right;
+    filter: alpha(opacity=0);
```

Comment:
`opacity=0` should be written with a single space on each side of the operator: `opacity = 0`